src/ServiceController.php

```diff
@@ -11,6 +12,12 @@
  */
 class ServiceController extends Controller
 {
+    const READ_REQUEST_SCOPE = 'read:holds';
```

Comment:
Are these are `holdrequests`? I think a `Hold` is what happens after a `HoldRequest` is successful. I'm just thinking ahead to when we actually have a `HoldsService`.